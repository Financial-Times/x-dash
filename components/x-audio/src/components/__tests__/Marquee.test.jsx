const { h } = require('@financial-times/x-engine');
const { shallow } = require('@financial-times/x-test-utils/enzyme');
const { Marquee } = require('../Marquee');

describe('Marquee', () => {
	test('should have necessary elements', () => {
		const props = {
			text: 'Moving text... Moving text...',
			className: 'class-name'
		};
		const subject = shallow(<Marquee {...props}/>);

		expect(subject).toMatchSnapshot();
	});
});
