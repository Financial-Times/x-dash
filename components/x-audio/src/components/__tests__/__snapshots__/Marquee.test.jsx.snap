// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Marquee should have necessary elements 1`] = `
<div
  className="class-name"
  style={
    Object {
      "overflow": "hidden",
    }
  }
>
  <span
    style={
      Object {
        "display": "inline-block",
        "transform": "translate3d(-0px, 0, 0)",
        "whiteSpace": "nowrap",
        "willChange": "transform",
      }
    }
    title="Moving text... Moving text..."
  >
    Moving text... Moving text...
  </span>
</div>
`;
